lib/ansible/plugins/filter/core.py

```diff
@@ -193,6 +193,10 @@ def regex_escape(string):
     '''Escape all regular expressions special characters from STRING.'''
     return re.escape(string)
 
+def from_yaml(input):
+    '''Load yaml and insure input value is always a str'''
+    return yaml.safe_load(str(input))
```

Comment:
There are several issues with this 'solution', encoding, typeerrors, etc. I would instead return the data if it is not a 'string_type' and convert if it is.